man/systemd.xml

```diff
@@ -70,8 +70,13 @@
       <varlistentry>
         <term><option>--test</option></term>
 
-        <listitem><para>Determine startup sequence, dump it and exit.
-        This is an option useful for debugging only.</para></listitem>
+        <listitem><para>Determine the initial start-up transaction (i.e. the list of jobs enqueued at
+        start-up), dump it and exit. This option is useful for debugging only. Note that during regular
+        service manager start-up further units might be started than this operation shows, because hardware,
```

Comment:
Hmm, "further units ... than this operation shows" feels ungrammatical somehow. What about "additional units not shown by this operation may be started"?